src/encoder.rs

```diff
@@ -1232,39 +1236,63 @@ pub fn encode_tx_block(
   fi: &FrameInvariants, fs: &mut FrameState, cw: &mut ContextWriter,
   w: &mut dyn Writer, p: usize, bo: &BlockOffset, mode: PredictionMode,
   tx_size: TxSize, tx_type: TxType, plane_bsize: BlockSize, po: &PlaneOffset,
-  skip: bool, bit_depth: usize, ac: &[i16], alpha: i16
-) -> bool {
+  skip: bool, bit_depth: usize, ac: &[i16], alpha: i16, for_rdo_use: bool
+) -> (bool, i64) {
     let rec = &mut fs.rec.planes[p];
     let PlaneConfig { stride, xdec, ydec, .. } = fs.input.planes[p].cfg;
 
     if mode.is_intra() {
       mode.predict_intra(&mut rec.mut_slice(po), tx_size, bit_depth, &ac, alpha);
     }
 
-    if skip { return false; }
+    if skip { return (false, 0); }
```

Comment:
Should this return an actual distortion instead of 0?

Reply:
As you know, if tx block is a part of skipped partition, then only pixel domain distortion is available hence later compute_tx_rd_cost() acknowledge it and compute distortion in pixel domain there. 
However, I will change it return to -1 here to make sure the distortion is not computed yet as has been signaled that way in other places in my patch. Thx for the comment!